### PR TITLE
Add max shot and aim inaccuracy to triggerbot

### PR DESCRIPTION
**Pull request by Vonr**
for now i dont see a reason for two settings there might be one but I'm too tired to look (there are 2 settings anyway)

### CODE REVIEW

**Comment by xAkiraMiura** [comment]
one is for movement inaccuracy and the other is for spread inaccuracy.
some weapons like the negev only suffer from movement inaccuracy so keeping them together would help these weapons

**Comment by Finndramere** [comment]
> one is for movement inaccuracy and the other is for spread inaccuracy.
> some weapons like the negev only suffer from movement inaccuracy so keeping them together would help these weapons

I think it doesn't work that way. Both have the same functionality one is limit to where aim will trigger and one is limit for when shot will be triggered(for autoshoot or while shooting). I think he's right that there's no point in having aim inaccuracy because well... You're not aiming when triggerbotting unless you implemented magnet trigger.  Correct me if I'm wrong.

from README.md
Max aim inaccuracy - maximum weapon inaccuracy allowing aimbot to run, lowering this value will e.g. disable aimbot while jumping or running.

File: Osiris/Hacks/Triggerbot.cpp [hunk 1]
```diff
@@ -80,7 +80,7 @@ void Triggerbot::run(UserCmd* cmd) noexcept
         if (float armorRatio{ weaponData->armorRatio / 2.0f }; activeWeapon->itemDefinitionIndex2() != WeaponId::Taser && HitGroup::isArmored(trace.hitgroup, trace.entity->hasHelmet()))
             damage -= (trace.entity->armor() < damage * armorRatio / 2.0f ? trace.entity->armor() * 4.0f : damage) * (1.0f - armorRatio);
 
-        if (damage >= (config.triggerbot[weaponIndex].killshot ? trace.entity->health() : config.triggerbot[weaponIndex].minDamage) && activeWeapon->getInaccuracy() <= config.triggerbot[weaponIndex].maxShotInaccuracy) {
+        if (damage >= (config->triggerbot[weaponIndex].killshot ? trace.entity->health() : config->triggerbot[weaponIndex].minDamage) && activeWeapon->getInaccuracy() <= config.triggerbot[weaponIndex].maxShotInaccuracy) {
```
Comment (playday3008) [inline Osiris/Hacks/Triggerbot.cpp]: config.triggerbot[weaponIndex].maxShotInaccuracy
you forgot "->" here

**Comment by Vonr** [comment]
rewriting on newer branch

**Pull request closed without merging**